crates/abd-clam/Cargo.toml

```diff
@@ -42,15 +45,7 @@ priority-queue = "1.3.2"
 
 # Only used in CHAODA
 # TODO: Break CHAODA out into an optional feature
-# automl = { version = "0.3.0", path = "../rust-automl" }
-# automl = { git = "https://github.com/nishaq503/rust-automl", branch = "updates/dependencies" }
```

Comment:
We no longer use `automl`. Only `smartcore`. In the future, I may switch out `smartcore` for `linfa` or some different crate if it is easier to use